system_tests/multigas_stylus_program_test.go

```diff
@@ -122,3 +122,58 @@ func TestMultigasStylus_AccountAccessHostIOs(t *testing.T) {
 		})
 	}
 }
+
+func TestMultigasStylus_EmitLog(t *testing.T) {
```

Comment:
Could you add a few test cases with different numbers of topics and other data sizes? It could be a test table.